src/controllers/user.controller.ts

```diff
@@ -189,3 +189,40 @@ export const patchMarkUserAdminHandler = async (
 		});
 	}
 };
+
+/**
+ * This controller will unsubscribe user marketing mail.
+ *
+ * @param req request
+ * @param res response
+ *
+ * @author arbaz23
+ */
+export const updateUnsubscribeUser = async (
+	req: Request<UpdateUnsubscribeUserSchema["params"]>,
+	res: Response,
+) => {
+	const { id } = req.params;
+
+	try {
+
+		if (!res.locals.user.receiveMarketingEmails) {
```

Comment:
```suggestion
		if (!res.locals.user.receive_marketing_emails) {
```



https://github.com/MultiEmail/backend/blob/e1adbb9e341189cde8cabfc8e17791650a3338b9/src/models/user.model.ts#L66